autoload/neomake/makers/ft/lhaskell.vim

```diff
@@ -0,0 +1,52 @@
+
+function! neomake#makers#ft#lhaskell#EnabledMakers()
+    " Have not tested hdevtools yet!
+    "return ['ghcmod', 'hdevtools', 'hlint']
+    return ['ghcmod', 'hlint'] 
```

Comment:
I think it's fine to enable `hdevtools` as well, as long as it claims to work for literate haskell code.